db/schema.rb

```diff
@@ -192,14 +192,14 @@
     t.integer "hb"
     t.integer "hbrentshortfall"
     t.integer "property_relet"
-    t.datetime "mrcdate", precision: nil
+    t.datetime "mrcdate"
```

Comment:
Any reason we're removing these?

Reply:
This happens automatically when I run migrations - I don't think this change actually makes a difference, I assume it's some sort of linting but I'm not actually certain why it's happening.